panel/io/compile.py

```diff
@@ -68,20 +68,15 @@ def setup_build_dir(build_dir: str | os.PathLike | None = None):
             shutil.rmtree(temp_dir)
 
 
-def check_cli_tool(tool_name: str) -> bool:
-    try:
-        cmd = shutil.which(tool_name)
-    except Exception:
-        cmd = None
-    if cmd:
-        return True
+def check_cli_tool(tool_name: str) -> str | None:
     if sys.platform == 'win32':
-        return shutil.which(f'{tool_name}.cmd')
-    return False
+        return shutil.which(tool_name) or shutil.which(f'{tool_name}.cmd')
+    else:
```

Comment:
only `shutil.which(tool_name)` seems sufficient even for windows
![image](https://github.com/user-attachments/assets/4d46d6af-855a-4062-9c4f-6c0dfdf02f42)

Reply:
I think this can differ based on the environment variable `PATHEXT`. I believe this is still correct and will not cause any harm. 

![image](https://github.com/user-attachments/assets/ea2c96fb-b127-4320-b247-fe3059ced939)